### ISSUE TITLE
(macOS) New Export - M4A Sample Rate exports as '0' above 48000Hz Sample Rate

### ISSUE THREAD

**Issue by chinakov** [audacity/audacity#4964]
### Bug description

If the User chooses any sample rate that is higher than 48000 Hz in the new Export UI with 'M4A (AAC) Files (FFmpeg)' as the format, the resulting file has '0' set as it's sample rate. 

If the User chooses any sample rate that is below OR 48000 Hz, the sample rate of the resulting file is correct and matches the setting chosen by the User during the export. 

### Steps to reproduce

1. Open Audacity
2. Add a Stereo track OR a Mono track
3. Generate audio OR import audio into the created track
4. Press File -> Export Audio...
5. Choose 'M4A (AAC) Files (FFmpeg)' as the Format for the export
6. Select anything higher than 48000 Hz as the sample rate
7. Export the audio
8. Open the audio file and check the sample rate
9. Observe Sample Rate to be '0'

### Expected behavior

According to @LWinterberg, everything up to 96000 Hz should be supported for the export. 

**This is how it is on Windows** - if the User selects anything above 96000 Hz, the sample rate will be limited to 96000 Hz in the result file, which confirms that >96000 Hz should be supported.

### Actual behavior

The resulting sample rate of the file is '0'.

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

macOS

### Additional context

This is a macOS only issue. 
macOS version: 13.2.1
Specs: Intel i5 Quad-Core, Radeo Pro 555 2GB, iMac (2017)

This issue is reproduced on Master branch as well.

**Comment by chinakov** [comment]
Clarification: the files can still be played, and in some programs, the sample rate is still shown. The '0' as the value for Sample Rate is only shown in one application called 'Meta' for editing metadata on macOS. 
Changing priority accordingly.

**Comment by petersampsonaudacity** [comment]
@chinakov 
Testing on W10 with @vsverchinsky 's latest branch build:  

I cannot repoduce this 
a) exported 44.1k stereo track at 96k
b) imported it back into the project
c) observed the imported track shows in its TCP 96000Hz
![image](https://github.com/audacity/audacity/assets/50205984/97299f15-04df-4dde-8f9b-707dc7e5b582)

**Comment by chinakov** [comment]
Yeah, @petersampsonaudacity  - it seems like this was only connected to a metadata viewing application for macOS. Audacity is not at fault here, 'Meta' is.

**Comment by petersampsonaudacity** [comment]
> Yeah, @petersampsonaudacity - it seems like this was only connected to a metadata viewing application for macOS. Audacity is not at fault here, 'Meta' is.

So does that mean that this issue can be closed?

**Comment by petersampsonaudacity** [comment]
@chinakov 

I also repeated the test on my Macbook Pro running 12.6.7 Monterey using @vsverchinsky 's latest branch build

Exporting a 44.1K stereo WAV at 384k - it also re-imported fine at 384k

**Comment by chinakov** [comment]
@petersampsonaudacity - yes, this is the issue I've also been able to reproduce. I guess it has something to do with the metadata, rather than the data exported itself.
